docs/content/dev/release_checklist.md

```diff
@@ -0,0 +1,195 @@
+# Release Checklist:
+
+## Preparations:
+
+- [ ] Create release PR(s)
+  - [ ] In the main repo (actually only create a local branch to include all the fixes, additions and so on and only push and create a PR after testing)
+  - [ ] Release PR in <https://github.com/hedgedoc/container>
+  - [ ] Release PR in <https://github.com/hedgedoc/social-media>
+- [ ] Security fixes (make sure all available undisclosed security fixes are merged)
+- [ ] Bump the version
+  - [ ] docs/content/dev/openapi.yml
+    - version number image
+  - [ ] docs/content/setup/manual-setup.md
+    - `git clone` branch
+    - `git checkout` branch
+  - [ ] package.json
+    - `version`
+  - [ ] Update docker-compose in the docs
+    - `docs/content/setup/docker.md`
```

Comment:
The checklist has the filename as the first place and then what to change in the file afterward for all former entries. I would do the same for this one.